File: In_Javascript/ex14.js
// Logika
function uchta_son(x,y,z) {
    if (x >= 20 && x < y || x < z) {
        return x;
    } else if (y >= 20 && y < x || y < z) {
        return y;
    } else if (z >= 20 && z < x || z < y) {
        return z;
    }
}

console.log(uchta_son(15,54,49));

File: In_Ruby/ex14.rb
# logika
def uchta_son(x,y,z)
    if x >= 20 && x < y || x < z
        return x
    elsif y >= 20 && y < x || y < z
        return y
    elsif z >= 20 && z < x || z < y
        return z
    end
end

puts uchta_son(15,54,49)